sql/database.sql

```diff
@@ -6423,6 +6455,7 @@ CREATE TABLE `users` (
   `fname` varchar(255) default NULL,
   `mname` varchar(255) default NULL,
   `lname` varchar(255) default NULL,
+  `suffix` varchar(255) default NULL,
```

Comment:
@ophthal
This is not in the upgrade script. Is this needed for the eye module (if not, then will postpone this)?